lib/src/util/prop_errors.dart

```diff
@@ -24,10 +24,10 @@ class PropError extends Error {
   static const String combinationPrefix = 'InvalidPropCombinationError: ';
 
   final String _messagePrefix;
-  final invalidValue;
+  dynamic invalidValue;
```

Comment:
This should stay final
```suggestion
  final dynamic invalidValue;
```